### PR TITLE
upgrade tests enable udf on ver > 2.1

### PR DESCRIPTION
**Pull request by knifewine**

### CODE REVIEW

File: upgrade_through_versions_test.py [hunk 1]
```diff
@@ -294,6 +294,11 @@ def setUp(self):
     def init_config(self):
         self.init_default_config()
 
+        if self.cluster.version() > '2.1':
+            enable_udf = {'enable_user_defined_functions': 'true'}
+            debug("Setting configuration option:\n" + pprint.pformat(enable_udf, indent=4))
+            self.cluster.set_configuration_options(values=enable_udf)
```
Comment (ptnapoleon) [inline upgrade_through_versions_test.py]: Does this potentially override other config options we want to preserve?

Comment (knifewine) [inline upgrade_through_versions_test.py]: Did a small test and looks like we're ok there

File: upgrade_through_versions_test.py [hunk 2]
```diff
@@ -294,6 +294,11 @@ def setUp(self):
     def init_config(self):
         self.init_default_config()
 
+        if self.cluster.version() > '2.1':
+            enable_udf = {'enable_user_defined_functions': 'true'}
+            debug("Setting configuration option:\n" + pprint.pformat(enable_udf, indent=4))
```
Comment (ptnapoleon) [inline upgrade_through_versions_test.py]: -1 to string concatenation with `+`.

**Comment by ptnapoleon** [comment]
+1 after you fix that debug statement

**Comment by knifewine** [comment]
That look ok to you?

**Comment by ptnapoleon** [comment]
Travis seems happy, and thus I am happy. +1

**Pull request merged**